hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/ozoneimpl/ContainerController.java

```diff
@@ -44,6 +47,8 @@
 
   private final ContainerSet containerSet;
   private final Map<ContainerType, Handler> handlers;
+  private static final Logger LOG =
+          LoggerFactory.getLogger(ContainerController.class);
```

Comment:
The indent of wrapped line is 4 space instead of 8.  This is a general code style of ozone.